composer.json

```
@@ -58,6 +58,9 @@
             "Yiisoft\\Yii\\Queue\\Tests\\": "tests"
         }
     },
+    "bin": [
+        "bin/queue"
```

```suggestion
        "bin/yii-queue"
```

It's global space, I suggest use prefix.

---

The idea when removing the console is to rename the package to yiisoft/queue, so queue makes sense.

---

I suggest rename utility only. Because it is global namespace